src/jit/utils.h

```diff
@@ -42,6 +42,22 @@ inline bool isPow2(T i)
     return (i > 0 && ((i - 1) & i) == 0);
 }
 
+inline bool isPow2f(float v)
+{
+    uint32_t i = *(uint32_t*)&v;
+    uint8_t exponent = (uint8_t)(i >> 23);
```

Comment:
You need to strip the sign bit

Reply:
@tannergooding  isn't it stripped by the uint8_t cast here?

Reply:
ah right. Just `double` needs fixing then.